.circleci/scripts/linux_test_sanity.sh

```diff
@@ -4,8 +4,8 @@ source scripts/install_utils
 
 MODELS=("fastrcnn" "fcn_resnet_101" "my_text_classifier_v2" "resnet-18" "my_text_classifier_scripted_v2" "alexnet_scripted" "fcn_resnet_101_scripted"
            "roberta_qa_no_torchscript" "bert_token_classification_no_torchscript" "bert_seqc_without_torchscript")
-MODEL_INPUTS=("examples/object_detector/persons.jpg,docs/images/blank_image.jpg" "examples/image_segmenter/fcn/persons.jpg" "examples/text_classification/sample_text.txt" "examples/image_classifier/kitten.jpg"
- "examples/text_classification/sample_text.txt" "examples/image_classifier/kitten.jpg" "examples/image_segmenter/fcn/persons.jpg" "examples/Huggingface_Transformers/QA_artifacts/sample_text.txt"
+MODEL_INPUTS=("examples/object_detector/persons.jpg,docs/images/blank_image.jpg" "examples/image_segmenter/persons.jpg" "examples/text_classification/sample_text.txt" "examples/image_classifier/kitten.jpg"
```

Comment:
If we include `deeplabv3_resnet_101` and `deeplabv3_resnet_101_scripted` models, we should also include the MODEL_INPUTS and the HANDLERS for those model in the same order that they are included in the listing so as to ensure that the sanity checks run successfully over those models.

Reply:
Postponed for the next iteration.